### DISCUSSION TITLE
The violin picture  of Seurat::VlnPlot seems not to be correct

### DISCUSSION THREAD

**Discussion by guoyangzou** [satijalab/seurat#5752] [Q&A]
many points were unable to be reflected by the violin picture plotted by Seurat::VlnPlot(Fig1). But these points were able to be reflected by the violin picture plotted by ggplot2::geom_violin(Fig2).

`my.sc %>% VlnPlot(features="ENSMMUG00000001980",split.by="cellType",assay="RNA",slot="data")+my.sc %>% VlnPlot(features="ENSMMUG00000001980",split.by="cellType",assay="RNA",slot="data",pt.size=0)`
<img width="1273" alt="企业微信截图_1647574544337" src="https://user-images.githubusercontent.com/74129206/158937159-be98cde2-54d5-4fbb-a342-01716a1482f6.png">
Fig1

`my.sc@assays$RNA@data["ENSMMUG00000001980",my.sc$cellType=="T cells"] %>% as_tibble %>% ggplot() + geom_violin(aes(x=1,y=value)) + geom_point(aes(x=1,y=value))`
<img width="1275" alt="企业微信截图_16475750217555" src="https://user-images.githubusercontent.com/74129206/158937833-2f708228-eb55-4d08-bc22-d30d608b5689.png">
Fig2

**Comment by timoast** [comment]
See https://github.com/satijalab/seurat/issues/5756
